firebase/package.json

```diff
@@ -41,6 +41,7 @@
     "@typescript-eslint/eslint-plugin": "^4.32.x",
     "@typescript-eslint/parser": "^4.32.x",
     "ajv": "^8.8.2",
+    "ajv-formats": "^2.1.1",
```

Comment:
Adds supporting for using `format: url` in Ajv schemaes

Reply:
It seems to be contained in the `json-schema` [specification](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.6)